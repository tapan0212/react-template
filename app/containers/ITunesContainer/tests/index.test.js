/**
 *
 * Tests for ITunesContainer
 *
 *
 */

import React from 'react';
import { renderProvider } from '@utils/testUtils';
// import { fireEvent } from '@testing-library/dom'
import { ITunesContainerTest as ITunesContainer } from '../index';

describe('<ITunesContainer /> container tests', () => {
  // let submitSpy

  beforeEach(() => {
    // submitSpy = jest.fn()
  });
  it('should render and match the snapshot', () => {
    const { baseElement } = renderProvider(<ITunesContainer />);
    expect(baseElement).toMatchSnapshot();
  });
});
