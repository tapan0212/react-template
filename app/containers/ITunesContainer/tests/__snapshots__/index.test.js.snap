// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ITunesContainer /> container tests should render and match the snapshot 1`] = `
<body>
  <div>
    <div>
      ITunesContainer
    </div>
  </div>
</body>
`;
